contrib/cirrus/logcollector.sh

```diff
@@ -32,6 +32,14 @@ case $1 in
     df) showrun df -lhTx tmpfs ;;
     ginkgo) showrun cat $CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log ;;
     journal) showrun journalctl -b ;;
+    varlink)
+       if [[ "$TEST_REMOTE_CLIENT" != "true" ]]
+       then
+          die 0 "No log, \$TEST_REMOTE_CLIENT is not 'true'"
+       else
+          showrun cat /tmp/varlink.log
```

Comment:
If you choose to remove the `tail` above, perhaps it could be moved here instead:
```
    showrun tail -10 /tmp/varlink.log
```
That way the full log is still preserved somewhere, and at least _something_ is available in case of error.

Reply:
okay.